modules/geolocation/unittests.py

```diff
@@ -202,6 +202,100 @@ def test_point_set_2(self):
         actual = self.locator.calculate_pixel_to_geo_mapping()
         # Test
         np.testing.assert_almost_equal(actual, expected)
+        
+        
+class TestOutputAnalysis(unittest.TestCase):
+    """
+    Tests Geolocation.outputanalysis(inputLocationTupleList)
+
+    """
+    def test_all_ones(self):
+        outputAnalysis = geolocation.OutputAnalysis()
+
+    # Setup
```

Comment:
Formatting nitpick: Comments should be at the same indentation level as the code around it.